src/super_gradients/scripts/export_recipe.py

```diff
@@ -0,0 +1,34 @@
+"""
+Script that saves a complete (i.e no inheritance from other yaml configuration files),
+ .yaml file that can be ran on its own without the need to keep other configurations which the original
+  file inherits from.
+
+Usage:
+    python export_recipe --config_name=cifar10_resnet save_path=/other/recipes/dir/my_complete_recipe.yaml -> saves cifar10_resnet_complete.yaml
+     in current working directory
+
+    python export_recipe --config_dir=/path/to/recipes/ config_name=my_recipe.yaml -> saves config_name_complete.yaml in current working directory
+
+    python export_recipe --config_dir=/path/to/recipes/ config_name=my_recipe.yaml save_path=/other/recipes/dir/my_complete_recipe.yaml
+     -> saves the complete recipe in /other/recipes/dir/my_complete_recipe.yaml
+
+:arg config_name: The .yaml config filename (can leave the .yaml postfix out, but not mandatory).
+
+:arg config_dir: The config directory path, as absolute file system path.
+ When None, will use SG's recipe directory (i.e path/to/super_gradients/recipes)
+
+:arg: The output path for the complete .yaml file.
+ When None, will use the current working directory.
+
+"""
+
+import argparse
+from super_gradients.common.environment.cfg_utils import export_recipe
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--config_dir", type=str, default=None, help="The config directory path")
```

Comment:
In convert_from_recipe I did a slightly different scheme, where first argument (config) is positional and mandatory. Second argument (output) is also positional but optional:

```
    parser.add_argument("config_name", type=str, help=".yaml filename")
    parser.add_argument("save_path", type=str, default=None, help="Destination path to the output .py file")
    parser.add_argument("--config_dir", type=str, default=pkg_resources.resource_filename("super_gradients.recipes", ""), help="The config directory path")
```

One still can use --config_name syntax, but it also enables use of command as follows: `convert_from_recipe.py coco2017_yolo_nas_s train_coco2017_yolo_nas_s.py` (Without --config_name).

Since second argument is optional, `convert_from_recipe.py coco2017_yolo_nas_s` would also work fine.

I suggest to add these changes to this script to make them consistent for the user.

Reply:
done f791e29886ed0dffb9b12db6ef0e7bd52ebe996e